libs/ngx-mime/src/lib/core/intl/help-intl.no_nb.ts

```diff
@@ -12,5 +12,6 @@ export class HelpIntlNoNb extends HelpIntl {
   line8 = '<strong>P</strong>: Går til forrige treff i objektet';
   line9 = '<strong>F</strong>: Fullskjerm av og på (Lukk med <strong>F</strong> eller <strong>ESC</strong>-tasten.)';
   line10 = '<strong>R</strong>: Rotér 90°';
+  line11 = '<strong>T</strong>:  Vis/skjul optisk gjenkjent tekst (for innhold som kan vise det)';
```

Comment:
This text does not match the english and the lithuanian translation according to [Google translate](https://translate.google.com/?source=osdd&sl=en&tl=no&text=Show%2Fhide%20optically%20recognized%20text%20(for%20content%20that%20can%20be%20displayed)&op=translate)
I also feel like the google translated version sounds better for the last part of the text.

Reply:
After discussing this with our product owner we agreed on the following change:

```suggestion
  line11 = '<strong>T</strong>:  Vis/skjul optisk gjenkjent tekst (for innhold som kan vises).';
```